package/kernel/linux/modules/fs.mk

```diff
@@ -31,11 +31,13 @@ define KernelPackage/fs-afs
   SUBMENU:=$(FS_MENU)
   TITLE:=Andrew FileSystem client
   DEFAULT:=n
-  DEPENDS:=+kmod-rxrpc +kmod-dnsresolver +kmod-fs-fscache
+  DEPENDS:=+kmod-rxrpc +kmod-dnsresolver +LINUX_6_6:kmod-fs-fscache +!LINUX_6_6:kmod-fs-netfs
   KCONFIG:=\
 	CONFIG_AFS_FS=m \
 	CONFIG_AFS_DEBUG=n \
-	CONFIG_AFS_FSCACHE=y
+	CONFIG_AFS_FSCACHE=y \
+	CONFIG_FSCACHE=y@ge6.12 \
+	CONFIG_FSCACHE_STATS=y@ge6.12
```

Comment:
The fscache support is compile into the `fs/netfs/netfs.ko` file of the `KernelPackage/fs-netfs` package. 

You should move these options to the `KernelPackage/fs-netfs` package.